src/app/login/components/LoginForm.tsx

```diff
@@ -123,7 +123,7 @@ export default function LoginForm() {
             </div>
 
             {isPending ? (
```

Comment:
`isPending` 조건에 따라 `loading`과 `disabled` 조건이 추가되는 부분만 다른 것 같은데, 아래와 같이 간결하게 작성할 수 있을 것 같습니다!

```suggestion
<div className='mt-[4rem] flex items-center justify-center'>
  <Button
    className='w-full'
    disabled={isDisabled}
    loading={isPending}
    onClick={handleSubmit}
    size='lg'
    variant='primary'
  >
    로그인
  </Button>
</div>
```